cider-velox/src/RawDataConvertor.cpp

```diff
@@ -463,10 +471,36 @@ RowVectorPtr RawDataConvertor::convertToRowVector(const CiderBatch& input,
   int num_cols = schema.getColumnCount();
   types.reserve(num_cols);
   columns.reserve(num_cols);
+  int inputColIndex = 0;
+
   for (int i = 0; i < num_cols; i++) {
     ::substrait::Type sType = schema.getColumnTypeById(i);
     types.push_back(getVeloxType(sType));
-    columns.push_back(toVeloxVector(types[i], sType, input.column(i), num_rows, pool));
+    auto currentData = input.column(i);
+    auto columNum = input.column_num();
+    if (sType.kind_case() == substrait::Type::kStruct) {
+      std::vector<VectorPtr> columnStruct{toVeloxVector(types[i]->childAt(0),
+                                                        sType.struct_().types(0),
+                                                        input.column(inputColIndex),
+                                                        num_rows,
+                                                        pool),
+                                          toVeloxVector(types[i]->childAt(1),
```

Comment:
For child type, it still can be struct type. We should add some test cases covering nested case (e.g., ```struct a {struct b { int c}}}```).

Reply:
@ZJie1 will it work if we convert the data just in flatten format and then set the schema to struct? At cider side I did in this way.

Reply:
I see that the data in the input is already flattened, so when the type is struct, I directly take two adjacent numbers.

Reply:
is there actually case that data in struct are more than two? The case here only fit for the  struct[data1, data2].

Reply:
> For child type, it still can be struct type. We should add some test cases covering nested case (e.g., `struct a {struct b { int c}}}`).

@yma11 what will cider do to process this case if we actually need handle this case ? how will the data stored in the `CiderBatch`

Reply:
Currently the data stored in CiderBatch is flattened but it should be different in velox vector as its different representation in memory? if so, it will be a complicated case but we need handle that.

Reply:
add TODO